.github/workflows/ci.yml

```diff
@@ -141,6 +141,8 @@ jobs:
         id: 'setup-monorepo'
         with:
           install: '${{ matrix.projectName }}...'
+          build: '${{ matrix.projectName }}'
```

Comment:
This change will slow down linting jobs in CI, reverting the previous optimization efforts.
Given that the necessary linting tools are already being installed, what's the use case for the building steps?